atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schema.java

```diff
@@ -402,4 +408,30 @@ public void addCleanupTask(String rawTableName, Supplier<OnCleanupTask> task) {
     public void ignoreTableNameLengthChecks() {
         ignoreTableNameLengthChecks = true;
     }
+
+    public SchemaMetadata getSchemaMetadata() {
+        // TODO (jkong): Memoize?
+        ImmutableSchemaMetadata.Builder builder = ImmutableSchemaMetadata.builder();
+
+        // This is kinda funky, but I can't really think of a better way
+        Map<TableReference, SchemaDependentTableMetadata> tableMetadatas =
+                Stream.of(tempTableDefinitions, tableDefinitions, indexDefinitions)
```

Comment:
Seems like we can remove `tempTableDefinitions` and the one place where it is accessed since it is always empty.

Reply:
(Removed in new version.)